### PR TITLE
[NUI] Add/change scroll event

### PR DESCRIPTION
**Pull request by neostom432**
### Description of Change ###
ScrollStartDragEvent      - Emitted when panning is start
ScrollEndDragEvent        - Emitted when panning is end
ScrollAnimationStartEvent - Emitted when scroll animation is start
ScrollAnimationEndEvent   - Emitted when scroll animation is end

### API Changes ###
NONE

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 1]
```diff
@@ -281,20 +281,39 @@ public class ScrollEventArgs : EventArgs
...
 
         /// <summary>
-        /// An event emitted when the scrolling starts, user can subscribe or unsubscribe to this event handler.<br />
+        /// An event emitted when the scrolling drag starts, user can subscribe or unsubscribe to this event handler.<br />
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
         /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public event EventHandler<ScrollEventArgs> ScrollStartedEvent;
+        public event EventHandler<ScrollEventArgs> ScrollStartDragEvent;
```
Comment (agnelovaz) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs]: Do you think ScrollDragStartEvent and ScrollDragEndEvent as more consistent with ScrollAnimationStartEvent and ScrollAnimationEndEvent?

Note:  Even although called Drag they will still be emitted with Flick.   Add in documentation?

Comment (neostom432) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs]: Change ScrollStartDragEvent/ScrollEndDragEvent to ScrollDragStartEvent/ScrollDragEndEvent .

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 2]
```diff
@@ -362,16 +381,28 @@ public override void OnChildRemove(View view)
             mScrollingChild = new View();
         }
 
-        private void OnScrollStart()
+        private void OnScrollDragStart()
+        {
+            ScrollEventArgs eventArgs = new ScrollEventArgs();
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs]: can I ask what kind of data are included in eventArgs ? it seems to new empty ScrollEventArgs object and set nothing.
then empty Args is sent to subscriber.

Comment (neostom432) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs]: For now, nothing needed. Just want to know when.

**Pull request merged**